lib/workers/branch/lerna.js

```diff
@@ -65,7 +65,7 @@ async function generateLockFiles(tmpDir, env) {
       logger.warn('Could not detect lerna in devDependencies');
     }
     logger.debug('Using lerna version ' + lernaVersion);
-    const cmd = `find . && ls -l && ${npmCmd} install && ${npxCmd} lerna@${lernaVersion} bootstrap -- --package-lock-only`;
+    const cmd = `find . && ls -l && ${npmCmd} install --package-lock-only --ignore-scripts && ${npxCmd} lerna@${lernaVersion} bootstrap -- --package-lock-only --ignore-scripts`;
```

Comment:
package-lock-only was missing in at the install cmd, so node_modules where installed for root

the npx lerna command causes some issues
- it seems like it is ignoring the package-lock-only parameter (it does work if i execute the same command manually)
- it seems like the progress output of lerna is piped to stderr, which results in a "false negative" for npm install `"stderr": "npx: Installierte 295 in 7.46s\nlerna info version 2.5.1\nlerna info Bootstrapping 8 packages\nlerna info lifecycle preinstall\nlerna info Installing external dependencies\nlerna info Symlinking packages and binaries\nlerna info lifecycle postinstall\nlerna info lifecycle prepublish\nlerna info lifecycle prepare\nlerna success Bootstrapped 8 packages\n"`

I am unsure how to pervent that, any suggestions?

Reply:
Is the extra `--` before ``package-lock-only` perhaps causing the problem?

I noticed the `stderr` as well, I'm not sure we can really fix that but I suppose can try to ignore it. Is it causing it to then error within the Renovate app too?

Reply:
BTW when I was seeing the `stderr` messages I was still getting the lock files I was after in the PRs I  linked to